Source/WTF/wtf/PlatformUse.h

```diff
@@ -307,7 +307,7 @@
 
 #if !defined(USE_TZONE_MALLOC)
 #if (CPU(ARM64) || CPU(X86_64)) && OS(DARWIN) && (__SIZEOF_POINTER__ == 8)
-#define USE_TZONE_MALLOC 0 /* temporarily disable FIXME rdar://140325747 */
+#define USE_TZONE_MALLOC 1 /* temporarily disable FIXME rdar://140325747 */
```

Comment:
Is this intentional? If so, you can remove this comment.

Reply:
This is intentional only for testing.  Still trying to figure out how to make this PR a Draft.  It's not ready for review